src/lib/middleware/package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Shouldn't this middleware have an appropriate name to indicate that it's for `tracing`?